android/app/src/main/java/com/zulipmobile/notifications/GCMPushNotifications.java

```diff
@@ -37,14 +37,14 @@
 
     public static final int NOTIFICATION_ID = 435;
     public static final String ACTION_NOTIFICATIONS_DISMISS = "ACTION_NOTIFICATIONS_DISMISS";
-    private LinkedHashMap<String, Pair<String, Integer>> conversations;
```

Comment:
Separately: how about specifically saying something like `ArrayList` here?

Reply:
Primarily I wrote List here because I think we might change it too LinkedList (details in the follow-up comment)